apps/www/components/ui/hover-border-gradient.tsx

```diff
@@ -55,7 +55,7 @@ export function HoverBorderGradient({
       }, duration * 1000);
       return () => clearInterval(interval);
     }
-  }, [hovered]);
+  }, [hovered, rotateDirection, duration]);
```

Comment:
the `HoverBorderGradient` component doesn't seem to be used anymore. Should we remove it entirely? cc @chronark